added state checkers for unknown state

RafAustralia opened the pull request:
added 3 separate code lines which will display the code for the U should not see me! unknown states in:  
Battery forced charge discharge cmd
Export Power Limit Mode
Ems Mode Selection

RafAustralia committed:
added state checkers for unknown state

added 3 separate code lines which will display the code for the U should not see me! unknown states in: 
Battery forced charge discharge cmd
Export Power Limit Mode
Ems Mode Selection

mkaiser commented on modbus_sungrow.yaml:
```diff
@@ -2181,7 +2181,7 @@ template:
             Stop (default)
           {% else %}
-            Unknown - should not see me!
+            Unknown - should not see me!  {{ (states('sensor.battery_forced_charge_discharge_cmd_raw') |int) }}
```
Please add a small "code:" before the number to make it more clear for the user (for all 3 else-branches). When you make a PR, I will merge it into the main.


``` Unknown - should not see me!  {{ (states('sensor.battery_forced_charge_discharge_cmd_raw') |int) }}```

``` Unknown - should not see me! Code: {{ (states('sensor.battery_forced_charge_discharge_cmd_raw') |int) }}```

RafAustralia committed:
unknown codes for batt charge discharge, export power limitation, EMS selection

added "code:"
for 
Battery forced charge discharge cmd
Export Power Limit Mode
Ems Mode Selection

RafAustralia reviewed (commented):
added code:
to the following
Battery forced charge discharge cmd
Export Power Limit Mode
Ems Mode Selection
